3_javascript/javascript2/week2/scripts.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Nice Work! Consider supporting lowercase for 2.7 as well. So for example it should also match this: "The three men and the pistol"

Reply:
> Nice Work! Consider supporting lowercase for 2.7 as well. So for example it should also match this: "The three men and the pistol"

@szoboszlaianna could u break it down for me not sure i get what u mean thanks..